src/main/java/org/apache/commons/lang3/CharUtils.java

```diff
@@ -174,9 +174,10 @@ public static char toChar(final Character ch, final char defaultValue) {
      * @return the char value of the first letter of the String
      * @throws IllegalArgumentException if the String is empty
      */
+    @SuppressWarnings("index:argument.type.incompatible") // #1: Validate.isTrue => string is non empty
     public static char toChar(final String str) {
```

Comment:
Instead of suppressing this warning, I would annotate `str` as `@MinLen(1) String`. This follows the principle of annotating normal behavior: https://checkerframework.org/manual/#annotate-normal-behavior